types/src/auction.rs

```diff
@@ -339,7 +339,7 @@ pub trait Auction:
     /// Accessed by: node
     fn run_auction(&mut self) -> Result<()> {
         if self.get_caller() != SYSTEM_ACCOUNT {
```

Comment:
According to Display for Error shouldn't those 2 cases be still `InvalidCaller?` 

> #[fail(display = "Function must be called by system account")]
> InvalidCaller = 16,